### PR TITLE
Docker update

### PR DESCRIPTION
**Pull request by mitche50**
- Add secrets/ folder to git ignore to have local storage of sensitive files
- modifications of scripts to run the correct docker scripts and pull correct config

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3.3'
 services:
     eth-approve-bot:
         image: eth-approve-bot
-        command: python -m scripts.rewards.eth.auto_approve_eth
+        command: python -m scripts.rewards.eth.approve_eth
```
Comment (SHAKOTN) [inline docker-compose.yaml:5]: Should we make different services for different chains? Like, can we add one more service for arb chain?

**Comment by SHAKOTN** [comment]
If we are going to use Docker configuration for development purposes, should we add `requirements-dev.txt` installation in this PR?

File: docker-compose.yaml (hunk 1 above)
Comment (mitche50) [reply docker-compose.yaml:5]: The Docker Compose is currently only used for running the approve bot on ETH.  All other bots are run by k8s and have specific YAML files that correspond with them in `https://github.com/Badger-Finance/badger-rewards/tree/main/.deploy/base`

**Comment by mitche50** [comment]
> If we are going to use Docker configuration for development purposes, should we add `requirements-dev.txt` installation in this PR?

We use the `Dockerfile` for our production bots spun up by k8s.  We could create a `Dockerfile.dev` and use this in tandem with `docker-compose start dev-service` to use for testing

**Review by SHAKOTN — APPROVED** [review]

**Pull request merged**